addons/dex/dex.yaml

```diff
@@ -26,6 +26,9 @@ spec:
   requires:
     - matchLabels:
         kubeaddons.mesosphere.io/provides: ingresscontroller
+    - matchLabels:
```

Comment:
dupe
```suggestion
```

Reply:
I believe we need this to match different addons:

https://github.com/mesosphere/kubernetes-base-addons/blob/f7d5de0f1be7249cc8f1f02c94047f14ae5ecf44/addons/gcpdiskprovisioner/gcpdiskprovisioner.yaml#L14-L18

Reply:
No, not a dup
There are two separate dependencies being matched.